channeldb/invoice_test.go

```diff
@@ -669,17 +675,33 @@ func TestQueryInvoices(t *testing.T) {
 		t.Fatalf("unable to make test db: %v", err)
 	}
 
-	// To begin the test, we'll add 50 invoices to the database. We'll
+	// To begin the test, we'll add 36 invoices to the database. We'll
 	// assume that the index of the invoice within the database is the same
 	// as the amount of the invoice itself.
-	const numInvoices = 50
+	const numInvoices = 36
 	var settleIndex uint64 = 1
 	var invoices []Invoice
 	var pendingInvoices []Invoice
+	var expiredInvoices []Invoice
 
 	for i := 1; i <= numInvoices; i++ {
 		amt := lnwire.MilliSatoshi(i)
-		invoice, err := randInvoice(amt)
+		var (
+			invoice *Invoice
+			err     error
+		)
+
+		// From every three consecutive invoices, make
+		// one expired and two non-expired.
```

Comment:
Since we start at `i := 1`, the second invoice will be expired, then every third invoice after that will be as well.